Symfony/CS/Fixer/Symfony/AliasFunctionsFixer.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Symfony;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Vladimir Reznichenko <kalessil@gmail.com>
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ */
+class AliasFunctionsFixer extends AbstractFixer
+{
+    /**
+     * @var string[] stores alias (key) - master (value) functions mapping
```

Comment:
Are you sure?
https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#824-var